bigchaindb/commands/bigchain.py

```diff
@@ -267,9 +266,9 @@ def run_add_replicas(args):
     try:
         add_replicas(conn, args.replicas)
     except (OperationError, NotImplementedError) as e:
-        logger.warn(e)
+        print(e)
     else:
-        logger.info('Added {} to the replicaset.'.format(args.replicas))
+        print('Added {} to the replicaset.'.format(args.replicas))
```

Comment:
Maybe as a note for later, for best practices or whatnot:

I think the commands should be explicit in always using `print()`, but the information that they print should also be represented in the logs somewhere. For example, here, we should use `print()` to tell the user we've added a node to the replicaset, but `add_replicas()` should handle logging that we've succeeded in adding to the replicaset.